pkg/programkind/programkind.go

```diff
@@ -205,6 +247,12 @@ func File(path string) (*FileType, error) {
 
 	// final strategy: DIY matching where mimetype is too strict.
 	s := string(hdr[:])
+	if bytes.Contains(hdr[:], []byte{'\x55', '\x50', '\x58', '\x21'}) {
```

Comment:
For readability, can you change this to `[]byte("UPX!")`?

Reply:
Updated in [`f849ff6` (#731)](https://github.com/chainguard-dev/malcontent/pull/731/commits/f849ff659f7acd25b7a2f71e2b643ea13b027088#diff-98d64e984a50bac5a4a8314d16140f66b05e118db774dc53208d9896d61352f9).